CMakeLists.txt

```diff
@@ -5,12 +5,16 @@ set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake-scripts)
 
 option(BUILD_EXAMPLES "Build example binaries" ON)
 option(BUILD_ALLEGRO4 "Build Allegro4 support" ON)
+option(USE_SSE "Use SSE instructions" ON)
```

Comment:
I am not sure how we to solve this in the best way, but maybe we can automatically disable SSE on non-x86/amd64-targets?